### DISCUSSION TITLE
Integrity Check

### DISCUSSION THREAD

**Discussion by ZampattiGilberto** [olahallengren/sql-server-maintenance-solution#468] [General]
Using the SQL Server DatabaseIntegrityCheck stored procedure we get a 2706 Error on a database; executing the DBCC CHECKDB command as it appears in the messages of the stored procedure, the command tells us that it is all right;
Is there anything we lost ?

**Comment by olahallengren** [comment]
What was the error message? Could you please post the complete output?
